test/linter/test-consistency.js

```diff
@@ -92,7 +100,7 @@ class ConsistencyChecker {
     // Test whether sub-features are supported when basic support is not implemented
     // For all unsupported browsers (basic support == false), sub-features should be set to false
     const unsupportedInParent = this.extractUnsupportedBrowsers(data.__compat);
-    /** @type {Partial<Record<string, [string, VersionValue][]>>} */
+    /** @type {?object.<string, string[]|VersionValue[]>} */
```

Comment:
This&nbsp;type is&nbsp;incorrect, as&nbsp;the&nbsp;`inconsistentSubfeaturesByBrowser` is never nullish.

Also, it&nbsp;doesn’t&nbsp;consist of&nbsp;entries that&nbsp;are&nbsp;either arrays&nbsp;of&nbsp;strings or&nbsp;arrays of&nbsp;VersionValues, but&nbsp;consists of&nbsp;arrays of&nbsp;arrays that&nbsp;have&nbsp;string and&nbsp;VersionValue as&nbsp;their&nbsp;entries.

```suggestion
    /** @type {object<string, Array<[string, VersionValue]> | undefined>} */
```